clients/web/src/stylesheets/layout/globalNav.styl

```diff
@@ -3,16 +3,13 @@ $emptyBorder=1px solid transparent
 $minHeight=500px
 
 .g-global-nav-main
-  box-shadow inset -1px 0 0 #eee
+  -webkit-box-shadow inset 0 -35px 45px -10px rgba(255, 255, 255, 1)
+  -moz-box-shadow inset 0 -35px 45px -10px rgba(255, 255, 255, 1)
+  box-shadow inset 0 -35px 45px -10px rgba(255, 255, 255, 1)
```

Comment:
Would it be simpler to set the color as `white`, `#fff`, `#ffffff` instead of `rgba(255, 255, 255, 1)`?